mortgageAPI.yaml

```diff
@@ -2394,6 +2394,22 @@ components:
           $ref: '#/components/schemas/AmountRange'
         increment:
           type: integer
+        amortizationType:
+          type: string
+          enum: [direct, indirect]
```

Comment:
Pls add an example. Although this seems kind of overkill, it generates better / easier to understand examples in the Swagger UI when viewing the Spec.

Reply:
done, see below